studio/src/pages/SettingsPage/AISettingsForm.tsx

```diff
@@ -220,16 +227,18 @@ export function AISettingsForm({
                                     to make requests to the{" "}
                                     {
                                       ProviderOptions[
-                                        providerField.value ?? "openai"
+                                        providerField.value as AiProviderType
                                       ]
                                     }{" "}
                                     API. It should be ignored by version control
                                     by default.
                                   </FormDescription>
                                   <FormControl>
                                     <ApiKeyInput
-                                      value={field.value ?? ""}
-                                      onChange={field.onChange}
+                                      value={(field.value as string) ?? ""}
+                                      onChange={(
```

Comment:
was this handler broken before? 😓

Reply:
I realize I need to revisit this in light of the settings model changes